javascript/javascript1/week3/TvSeries.js

```diff
@@ -0,0 +1,69 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0
+  },
+  {
+    title: "Friends",
+    days: 6,
+    hours: 15,
+    minutes: 32
+  }
+];
+
+const daysInYear = 365.2425;
```

Comment:
i like your usage of these constants. A good naming convention for constants is to use all caps. DAYS_IN_YEAR, HOURS_IN_DAY, etc. That way in the other parts of the code you know that noone is changing them.